libs/core/device/constants/device-id.ts

```diff
@@ -3,5 +3,4 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-export * from "./cache-data.listener"
-export * from "./data-sync.listener"
+export type DeviceId = string // string as serialNumber or self-generated value when empty
```

Comment:
Certainly, the decision to create a type to mask an existing type can vary based on team practices. I'm open to discussing it further during our sync meeting since I don't have a strong stance on this matter.